srcpkgs/forgejo/template

```diff
@@ -0,0 +1,70 @@
+# Template file for 'forgejo'
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# - [Installation from source](https://forgejo.org/docs/next/contributor/from-source)
+# - [Forgejo v9.0 is GPLv3+](https://codeberg.org/forgejo/forgejo/commit/94631c)
+
+homepage="https://forgejo.org"
+
+# https://github.com/void-linux/void-packages/blob/master/Manual.md#available_vars
+
+# Forgejo accepts contributions compatible with the GPLv3-or-later license.
+license="GPL-3.0-or-later"
+
+maintainer="Markus Heiser <markus.heiser@darmarit.de>"
+pkgname=forgejo
+# https://forgejo.org/docs/next/contributor/release/#release-cycle
+version=9.0.0
+revision=1
+_version="$(echo "${version}" | tr '.' '_')"
+short_desc="Forgejo is a self-hosted lightweight software forge"
+
+# go-bindata is from commit (dated in Oct 23, 2015):
+# - https://github.com/jteeuwen/go-bindata/commit/a0ff2567cf
+# - https://github.com/void-linux/void-packages/blob/059fd3968/srcpkgs/go-bindata/template#L5
+# go-bindata/go-bindata is more updated compared to jteeuwen/go-bindata
+# - https://github.com/go-bindata/go-bindata/releases
+
+hostmakedepends="go-bindata"
+makedepends="sqlite-devel pam-devel"
+depends="git git-lfs bash"
```

Comment:
Does this really depend on `bash`? Is `git-lfs` strictly required?

Reply:
From the installation description of forgejo [ref](https://forgejo.org/docs/latest/admin/installation-binary/):

> Make sure `git` and `git-lfs` are installed on your system.

And for the user creation they use: `useradd --system --shell /bin/bash` further I read:

> Forgejo will run as that user, and when accessing git through SSH (which is the default)

*.. this shell is used.* .. Not sure these SSH tasks do need a bash, but a shell is needed .. and I want to stay with the official installation instruction.

Reply:
We should not prefer or require bash when it isn't strictly required, because some people may wish to run installations without bash.